UI/ClientUI.cpp

```diff
@@ -651,6 +656,11 @@ ClientUI::ClientUI() :
 
     // Set the root path for image tags in rich text.
     GG::ImageBlock::SetDefaultImagePath(ArtDir().string());
+
+    // Set facet to format timestamp in chat.
+    boost::posix_time::time_facet* facet = new boost::posix_time::time_facet();
+    facet->format(ClientUI::DisplayTimestampFormat().c_str());
+    m_date_format_sstream.imbue(std::locale(std::locale(""), facet));
```

Comment:
does this take ownership of `facet` or will this leak?

Reply:
Yes, it takes ownership https://github.com/freeorion/freeorion/pull/1813#pullrequestreview-69426185

Reply:
I thought I'd already asked about that, but didn't see the comment in this pull request.

Reply:
It was in the base PR.